adoptopenjdk-api-v3-frontend/src/main/kotlin/net/adoptopenjdk/api/v3/routes/info/AvailableReleasesResource.kt

```diff
@@ -16,16 +14,14 @@ import javax.ws.rs.core.MediaType
 @Tag(name = "Release Info")
 @Path("/v3/info/")
 @Produces(MediaType.APPLICATION_JSON)
-@Timed
 @ApplicationScoped
-@GZIP
 class AvailableReleasesResource
 @Inject
 constructor(
     private val apiDataStore: APIDataStore
 ) {
     @GET
-    @Path("/available_releases/")
+    @Path("/available_releases")
```

Comment:
RESTEasy classic ignores trailing slashes in `@Path`s.
RESTEasy Reactive has little changes in behavior: for `@Path` with trailing slashes, a request without trailing slash leads to a 404.
BUT: `@Path` without trainling slashes respond to requests with as well as without trailing slashes.

So with this change, the application behaves like before.
Did report this to Quarkus: https://github.com/quarkusio/quarkus/issues/16406 (ticket contains also my tests with this app), it's not a problem at all, put the Quarkus Team possibly would discuss the topic.